### PR TITLE
chore: renaming master branch to development

### PR DESCRIPTION
**Pull request by artursudnik**
This PR implements changes needed to rename the `master` branch to the `development` branch.

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: Build and deploy
 
 on:
   push:
-    branches: ['master']
+    branches: ['development']
```
Comment (royki) [inline .github/workflows/deploy.yml]: According to your requirement it should be `develop`. or you want to make the branch name `development` ?

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -31,7 +31,7 @@ jobs:
         id: env_vars
         run: |
           echo "Running on branch ${{ github.ref }}"
-          if [ "${{ github.ref }}" = "refs/heads/master" ]; then
+          if [ "${{ github.ref }}" = "refs/heads/development" ]; then
```
Comment (royki) [inline .github/workflows/deploy.yml]: same as above.

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (artursudnik) [reply .github/workflows/deploy.yml]: I apologize. It is going to be `development`.

File: .github/workflows/deploy.yml (hunk 2 above)
Comment (artursudnik) [reply .github/workflows/deploy.yml]: I apologize. It is going to be `development`. I am editing the requirement.

**Review by royki — APPROVED** [review]

**Pull request merged**